packages/api/cms-api/src/page-tree/pages-entity-info.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from "@nestjs/common";
+
+import { EntityInfoServiceInterface } from "../dam/files/decorators/entity-info.decorator";
+import { DocumentInterface } from "../document/dto/document-interface";
+import { PageTreeReadApiService } from "./page-tree-read-api.service";
+
+@Injectable()
+export class PagesEntityInfoService implements EntityInfoServiceInterface<DocumentInterface> {
```

Comment:
Remember that there may be other document types that are not pages, but could use the same info service. I'd call it PageTreeNodeEntityInfoService or PageTreeNodeDocumentEntityInfoService

Reply:
I renamed it to `PageTreeNodeDocumentEntityInfoService`

But that basically also means that the name `getFirstNodeByAttachedPageId()` because it works with all documents (not just pages)